wiki/en/Client-Troubleshooting.md

```diff
@@ -70,7 +70,7 @@ You can set your "New Client Level" to a low value (e.g. 10), or set the musicia
 
 ### Can't see the Server you want to join?
 
-First check that you have the right genre Server selected in your Connect window. Sometimes network issues mean your client won't list all the available Servers. If you know the name of the Server you want to join, you can [look up its IP address here](https://explorer.jamulus.io/). Enter the IP address in the "Server Name/Address" field in the Connect Setup window to connect to it.
+First check that you have the right genre Server selected in your Connect window. Sometimes network issues mean your Client won't list all the available Servers. If you know the name of the Server you want to join, you can [look up its IP address here](https://explorer.jamulus.io/). Enter the IP address in the "Server Name/Address" field in the Connect Setup window to connect to it.
```

Comment:
![image](https://user-images.githubusercontent.com/1549463/156407067-e46300bc-e23a-4c5b-8a4d-304939273274.png)
> Enter the IP address in the "Server Name/Address" field

That's still wrong...  It really needs fixing.  It doesn't need to be an IP address and the field is called "Server Address" (based on a definition that should state it's the host name or IP number along with an optional port number, joined by a colon).

That's not what you were changing, of course... still needs doing, it's been annoying me for ages...